lib/internal/loader/index.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const Loader = require('internal/loader/Loader');
+
+const loader = new Loader();
+
+exports = module.exports = {
+  loader,
```

Comment:
we could move it to be a static ref since these files are never exposed to user code.